### ISSUE TITLE
Migrate to the new B2ACCESS URL

### ISSUE THREAD

**Issue by apw1388** [EUDAT-B2SHARE/b2share#1449]
Please make the B2ACCESS base_url configurable. With upcoming changes the whole software stack must reinstalled.

**Comment by nharraud** [comment]
The B2ACESS is already configurable via the config. However the documentation needs some improvement as to how to do it. I will create another issue for that.

Could you please explain what are the upcoming changes?

**Comment by apw1388** [comment]
We will switch to port from 8443 to 443 because users are often blocked by firewalls, if they use WiFi. The port in EntityID will be removed.

**Comment by nharraud** [comment]
Ok thank you for the information. I have two questions:
- When is this change planned?
- Is it possible to have a transition period during which both ports are accepted? Otherwise login to B2SHARE.eudat.eu will not work.

We will have to
- [ ] make an announcement explaining how and when to change the configuration.
- [ ] change the default configuration.
- [ ] republish a docker container.

@thijscobben @emanueldima @SarahBA @Tlacahuepan

**Comment by apw1388** [comment]
At the moment we have no date for it. The ports should be no problem. We have planed to inform all SP admins and users before we do the changes.
